package.json

```diff
@@ -8,6 +8,7 @@
     "start": "nx serve",
     "start:ptg-angular": "nx serve ptg-angular-app",
     "start:ptg-react": "nx serve ptg-react-app",
+    "start:pool-tracker": "nx serve pool-tracker",
```

Comment:
Can you add build command as well ? similar to other apps

Reply:
Added build command in react commit, please check